src/docs/stan-reference/functions.tex

```diff
@@ -3007,6 +3007,20 @@ \subsection{Matrix Exponential}
 \fitem{matrix}{matrix\_exp}{matrix \farg{A}}{
 The matrix exponential of \farg{A}}
 %
+
+%
+\fitem{matrix}{matrix\_exp\_multiply}{matrix \farg{A}, matrix \farg{B}}{
+The multiplication of matrix exponential of \farg{A} and
+matrix \farg{B}; algebraically equivalent to the less efficient form
+\code{matrix\_exp(\farg{A}) * \farg{B}}.}
+%
+
+%
+\fitem{matrix}{scale\_matrix\_exp\_multiply}{real \farg{t}, matrix \farg{A}, matrix \farg{B}}{
+The multiplication of matrix exponential of \farg{t}\farg{A} and
+matrix \farg{B}; algebraically equivalent to the less efficient form
```

Comment:
[optional]

This should really be `\farg{t} * \farg{A}` in the less efficient form.  I'll approve and leave it up to you whether to fix it or not.